app/components/Settings/Preview/Preview.react.js

```diff
@@ -201,7 +200,7 @@ class Preview extends Component {
             * lastSuccessfulQuery is the result of the last successful query
             * and should have the form {rows:[[]], columnnames:[]}
             */
-            if( !has('error', content) && has('rows', content) && has('columnnames', content) ){
+            if ( !has('error', content) && has('rows', content) && has('columnnames', content) ) {
```

Comment:
while this leading space after `if` is correct, I'm surprised that eslint didn't fix the space before `( !has`. Shouldn't this be `if (!has` rather than `if ( !has`?

Reply:
@chriddyp you're right, `if (!has` is way more common, hence nicer.

Problem was that [space-in-parens](https://eslint.org/docs/rules/space-in-parens) wasn't activated (it isn't by default, which is a bit surprising). I'll create a pull request soon that activates the rule and fixes the respective errors.